StringList/StringList.cpp

```diff
@@ -0,0 +1,357 @@
+#include "stringlist.h"
+#pragma warning(disable: 4996)
+
+#include <stdlib.h>
+#include <string.h>
+
+void StringListInit(char*** list)
+{
+    // creating dummy node
+    *list = static_cast<char**>(malloc(sizeof(char**) * 2));
```

Comment:
malloc returning a pointer or null. you need check it

Reply:
It is checked in line 12:
if (*list == nullptr)
        exit(-1);